src/components/Claim/Link/FlowManager.tsx

```diff
@@ -23,10 +26,23 @@ const FlowManager = ({
         SUCCESS:
             recipientType != 'iban' && recipientType != 'us'
                 ? onchainViews.SuccessClaimLinkView
-                : offrampViews.SuccessClaimLinkIbanView,
+                : OfframpSuccessView
```

Comment:
Thought: the bot has a good point on readability (or a comment would prob do the trick too).

Apart from that it might be the case that in the future we might have to deal with more than 2 different success/confirm screens, so this part might have to be refactored a bit in the future

Also right now

Reply:
Agree for readability. Could also extract the inline conditional into a small utility redirect function like `getOfframpView(viewType, recipientType)`:
```ts
{
        INITIAL: InitialClaimLinkView,
        CONFIRM: getOfframpView("confirm", recipientType),
        SUCCESS: getOfframpView("success", recipientType)
}
```
It's actually personal preference, i like using ifs more than inline a lot of the time